### PR TITLE
EOS-15360: Unboxing: Do cluster health validation at the end of unboxing

### PR DESCRIPTION
**Pull request by 83bhp**

### CODE REVIEW

File: cli/src/factory_ops/unboxing/init [hunk 1]
```diff
@@ -397,6 +394,8 @@ do
 done
 echo -e "Ok.\n"
 
+echo "Ensuring the cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
+ensure_healthy_cluster
```
Comment (mssawant) [inline cli/src/factory_ops/unboxing/init:398]: where do we run `pcs resource cleanup`?

**Review by ipoddubnyy — APPROVED** [review]
looks good to me

File: cli/src/factory_ops/unboxing/init (hunk 1 above)
Comment (mssawant) [reply cli/src/factory_ops/unboxing/init:398]: okay we already do that before, found it.

Comment (83bhp) [reply cli/src/factory_ops/unboxing/init:398]: it's there at line# 366

Comment (mssawant) [reply cli/src/factory_ops/unboxing/init:398]: If not already logged, kindly log output of pcs status before and after the call to ensure_healthy_cluster. Thank you.

**Review by mssawant — APPROVED** [review]
Minor comment.

**Review by ujjwalpl — APPROVED** [review]

**Review by mssawant — APPROVED** [review]

File: cli/src/factory_ops/unboxing/init (hunk 1 above)
Comment (83bhp) [reply cli/src/factory_ops/unboxing/init:401]: done.

**Review by ypise — APPROVED** [review]

**Review by mssawant — APPROVED** [review]

**Pull request merged**